studio/.gitignore

```diff
@@ -33,3 +33,5 @@ yarn-error.log*
 .env.production.local
 
 .vercel
+
+/public/dashboard
```

Comment:
```suggestion
```

Reply:
thanks !

Reply:
@MildTomato didn't you know @joshenlim is telepathic?

Reply:
this was weirdly creepy